app/javascript/controllers/file_auth_controller.js

```diff
@@ -16,7 +16,7 @@ export default class extends Controller {
         this.displayAccessTokenError(event.data)
       } else {
         this.cacheToken(event.data.accessToken, event.data.expiresIn)
-        window.dispatchEvent(new CustomEvent('login-success', { detail: {} }))
+        window.dispatchEvent(new CustomEvent('show-message-panel', { detail: {} }))
```

Comment:
is this line still needed? will the new one take care of this?

Reply:
Still needed. The new line will not take care of it. This line happens after a user has hit the login button. The other location happens as soon as the user loads the page. They both need to exist.